data-models/protobufs/maintenance_request.proto

```
@@ -0,0 +1,72 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package datamodels;
+
+import "google/protobuf/timestamp.proto";
+import "health_event.proto";
```

Buf (1.72.0)

[error] imported file does not exist

(COMPILE)